aws/cloudformation/ecs/posthog.yaml

```diff
@@ -724,6 +716,12 @@ Resources:
               Value: !Ref 'DefaultFromEmail'
             - Name: SENTRY_DSN
               Value: !Ref 'SentryDSN'
+            - Name: IS_BEHIND_PROXY
+              Value: 'true'
+            - Name: TRUST_ALL_PROXIES
+              Value: 'true'
```

Comment:
@fuziontech ideally we would set the ELB's IP address using the `TRUSTED_PROXIES` env var instead of trusting all proxies, can we do this?

Reply:
We probably could, but I don't think ELB ip's are sticky so it might work at first and then break later. This is probably good enough for now